src/templates/homepage/HeroSection.jsx

```diff
@@ -101,9 +143,21 @@ const KeyHighlightElem = ({ title, description }) => (
 )
 
 const KeyHighlights = ({ highlights }) => (
-  <section id="key-highlights" className="bp-section is-paddingless">
-    <div className="bp-container">
-      <div className="row is-gapless has-text-centered">
+  <section
+    id={editorStyles["key-highlights"]}
+    className={getClassNames(editorStyles, ["bp-section is-paddingless"])}
+  >
+    <div
+      id="key-highlights"
+      className={getClassNames(editorStyles, ["bp-container"])}
+    >
```

Comment:
not sure if this will impact e2e but the `section` and and the `div` have different ids after compared to before

Reply:
I've verified that e2e tests still work! The `key-highlights` inserted in the div is to make up for `key-highlights` no longer being useable in the section